### PR TITLE
Update react to 0.13

### PR DESCRIPTION
**Pull request by aweiksnar**
Dependency upgrade to react 0.13.2

### CODE REVIEW

File: app/lib/draggable.cjsx [hunk 1]
```diff
@@ -7,7 +7,6 @@ module.exports = React.createClass
   _previousEventCoords: null
 
   propTypes:
-    children: React.PropTypes.component.isRequired
```
Comment (brian-c) [inline app/lib/draggable.cjsx]: Is something here deprecated now?

**Comment by brian-c** [comment]
@aweiksnar Mind bumping coffee-rectify to 3.0.0 while you're in there?

I'll take care of that getDOMNode-in-render warning next time I update the classifier.

File: app/lib/draggable.cjsx (hunk 1 above)
Comment (aweiksnar) [inline app/lib/draggable.cjsx]: Yeah looks like it's no longer in the valid prop-types https://facebook.github.io/react/docs/reusable-components.html

**Comment by aweiksnar** [comment]
Just bumped it--sorry totally missed that warning

**Comment by brian-c** [comment]
Thanks.

The warning will take some work to fix. I think that's where it's trying to find the best position for the drawing tool details tooltip, and I'm not really happy with how that's working anyway.

cc: @saschaishikawa; we were talking about updating React the other day.

File: app/lib/draggable.cjsx (hunk 1 above)
Comment (aweiksnar) [inline app/lib/draggable.cjsx]: Ah I guess it's just React.PropTypes.element now. I'll get that in the next one

**Pull request merged**